leader-election-api/src/main/java/com/palantir/paxos/PaxosValue.java

```diff
@@ -39,6 +39,9 @@
     @Nullable
     final byte[] data;
     final String leaderUuid;
+    @Nullable
+    final String leaderUrl;
```

Comment:
Probably better for offline discussion: I'm not comfortable with adding this to PaxosValue, the concept of a URL doesn't quite exist at this level.